### ISSUE TITLE
Indicate dynamic change of watch variables to screen reader users

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#49786]
Sighted users have access to dynamic variable state, screen reader users do not. We should announce watch dynamic variable changes.

Consider if just variable name or variable name + value should be announced
